version/version.go

```diff
@@ -0,0 +1,10 @@
+package version
+
+const (
+	// Version is the Operator's version
+	Version = "0.11.0-beta.2"
```

Comment:
IMHO this should be on the version bump manual too.

Reply:
Let's discuss this out of band and leave this out of the context of this PR.